.github/validation_samples/reduced_ldmx/config.py

```diff
@@ -111,7 +111,9 @@
 rSeedTracking.layer12_midpoint = layer12_mid
 rSeedTracking.layer23_midpoint = layer23_mid
 rSeedTracking.layer34_midpoint = layer34_mid
+rSeedTracking.recoil_uncertainty = [0.006, 0.085]
 rSeedTracking.ecal_distance_threshold = 15.0
+rSeedTracking.buildHistograms()
```

Comment:
```suggestion
rSeedTracking.title = ""
rSeedTracking.buildHistograms()
```
if we just want to resolve the issue